roles/promtail/files/config.yml

```diff
@@ -10,24 +10,25 @@ clients:
 # hardcoded, gotta replace by var in ansible playbook and lineinfile
  
 scrape_configs:
-
-#- job_name: system
-#  static_configs:
-#  - targets:
-#      - localhost
-#    labels:
-#      job: varlogs
-#      __path__: /var/log/*log
-
-# no logs needed
-
 - job_name: journal
   journal:
     json: false
     max_age: 12h
     path: /var/log/journal
     labels:
+      #<custom label>
       job: systemd-journal
   relabel_configs:
     - source_labels: ['__journal__systemd_unit']
       target_label: 'unit'
+    - source_labels: ['__journal_syslog_identifier']
+      target_label: 'syslog_identifier'
+    - source_labels: ['__journal__hostname']
+      target_label: 'hostname'
+    - source_labels: ['__journal_priority']
+      target_label: 'log_level'
+  pipeline_stages:
+  - template:
+      source: log_level
+      template: '{{ if eq .Value "0" }}{{ Replace .Value "0" "emerg" 1 }}{{ else if eq .Value "1" }}{{ Replace .Value "1" >  - labels:
+      log_level:
```

Comment:
Still missing requested 2w retention period. Given the log rate this is not a luxury, absolute requirement before we go live!!!

Reply:
changed in the latest commit
https://github.com/ipfs-search/ipfs-search-deployment/commit/0df2ecf01c22ce909362e8374859d57a4d5a6fd4

has been configured on Loki side